src/garage/torch/policies/continuous_nn_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
+This modules creates a continuous policy network.
+
+A continuous neural network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+import akro
+import torch
+
+from garage.tf.policies.base import Policy
+
+
+class ContinuousNNPolicy(Policy):
+    """
+    Implements a module-agnostic policy network.
+
+    The policy network selects action based on the state of the environment.
+    It uses a PyTorch neural network module to fit the function of pi(s).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 policy_type='Deterministic',
+                 input_include_goal=False):
+        """
+        Initialize class with multiple attributes.
+
+        Args:
+            env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+            nn_module (nn.Module): Neural network module in PyTorch.
+            policy_type (str): Type of policy, deterministic or stochastic.
+        """
+        assert isinstance(env_spec.action_space, akro.Box)
+
```

Comment:
you need to validate `policy_type` here and raise a ValueError if it's not a valid type.

Reply:
I really think it's easier to just not have `policy_type` be a thing.